webapp/src/containers/SubscribeModal/index.tsx

```diff
@@ -226,6 +240,16 @@ const SubscribeModal = () => {
                         />
                     ))
                 }
+                {
+                    isAnyProjectLinked && showResultPanel &&
```

Comment:
this change was not working fine, I've fixed here https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/63/files#diff-020aea5bc738fb71db78ff41f22a03418982aa42e1fa9d686e5cd5f6e3058d5e

you can see for the betterment